dts/bindings/gpio/st,stm32-gpio.yaml

```diff
@@ -5,7 +5,11 @@ description: STM32 GPIO node
 
 compatible: "st,stm32-gpio"
 
-include: [gpio-controller.yaml, base.yaml]
+include:
+  - name: gpio-controller.yaml
+    property-blocklist:
+      - ngpios
```

Comment:
It doesn't seem necessary to do that. See the example provided by @erwango in nxp,pcal6416a.yaml, I don't see a property-blocklist (and it presumably works).

Reply:
Since that used `const`/`required`, it basically just added a `const` to the property binding. However, if there was a tree that used it (I actually dont see this driver in any board/shield), it would still need to be in the tree; its just anything other than `ngpios = <8>;` would be an error (missing it would be a violation of `required` and any other value a violation of `const: 8`)